source/default/_types/particle-element.ts

```diff
@@ -0,0 +1,13 @@
+export default class ParticleElement {
+    constructor(
+        public name: string
+    ) {}
+
+    enable(): any {
```

Comment:
Can the return value from enable and disable be narrowed down further even to a union type? I am not super familiar with the implementation details here.

Reply:
changing it to void, good call